src/Controller/CRUDController.php

```diff
@@ -212,7 +212,7 @@ public function batchActionDelete(ProxyQueryInterface $query)
      *
      * @return Response|RedirectResponse
      */
-    public function deleteAction($id)
+    public function deleteAction($id) // NEXT_MAJOR: Remove the unused $id parameter
```

Comment:
No deprecations here?

Reply:
@core23 there is no default value. So if I make a deprecation, the dev can’t fix it correctly in his code.

Reply:
Can't you add a default value to fix this? AFAIK this is BC.

Reply:
@core23  https://github.com/sonata-project/SonataAdminBundle/pull/5807#discussion_r360949215

Reply:
Thanks